include/hpp/pinocchio/fwd.hh

```diff
@@ -29,6 +29,8 @@
 # include <hpp/fcl/math/matrix_3f.h>
 # include <pinocchio/multibody/model.hpp>
 
+# include <hpp/pinocchio/deprecated.hh>
```

Comment:
Did you possibly forget to add in the git repo include/hpp/pinocchio/deprecated.hh